nservicebus/transactional-session/index.md

```diff
@@ -140,19 +169,31 @@ Internally the transactional session doesn't use a single transaction that spans
 4. The user can execute any required message operations using the transactional session.
 5. The user can store any data using the persistence-specific session, which is accessible through the transactional session.
 6. When all operations are registered, the user calls ´Commit´ on the transactional session.
-7. A control message is dispatched to the local queue.
+7. A control message to complete the transaction is dispatched to the local queue. The control message is independent of the message operations and is not stored in the outbox record.
```

Comment:
"to complete the transaction" is confusing since the transaction in this context is the storage transaction

```suggestion
7. A control message to trigger the dispatch mechanism is sent to the local queue. The control message is independent of the message operations and is not stored in the outbox record.
```

Reply:
We discussed it and are keeping the current wording